src/drivers/mpu6500/mpu6500.cpp

```diff
@@ -753,7 +753,7 @@ int MPU6500::reset()
 	_gyro_range_scale = (0.0174532 / 16.4);//1.0f / (32768.0f * (2000.0f / 180.0f) * M_PI_F);
 	_gyro_range_rad_s = (2000.0f / 180.0f) * M_PI_F;
 
-	set_accel_range(8);
```

Comment:
@bkueng is this an OK system wide change or does it need to be driven from a board_config.h ?